sys/compat/linuxkpi/common/include/net/mac80211.h

```diff
@@ -886,7 +886,7 @@ struct ieee80211_ops {
 	void (*sta_set_4addr)(struct ieee80211_hw *, struct ieee80211_vif *, struct ieee80211_sta *, bool);
 	void (*sta_set_decap_offload)(struct ieee80211_hw *, struct ieee80211_vif *, struct ieee80211_sta *, bool);
 
-	u64  (*prepare_multicast)(struct ieee80211_hw *, struct netdev_hw_addr_list *);
+	uintptr_t  (*prepare_multicast)(struct ieee80211_hw *, struct netdev_hw_addr_list *);
```

Comment:
I suggest splitting the last commit to have a separate commit for the s/u64/uintptr_t/ for the multicast changes vs the changes to add various subobject bounds annotations.

Reply:
Splitted.

Reply:
Thanks, maybe use "linux wifi multicast:" in the tag for this commit?

Reply:
Reworded!